lib/rules/inline-link-to.js

```diff
@@ -5,6 +5,16 @@ const Rule = require('./base');
 const message = 'The inline form of link-to is not allowed. Use the block form instead.';
 
 module.exports = class InlineLinkTo extends Rule {
+  static get meta() {
+    return {
+      description: 'disallows the inline form of `link-to`',
+      category: 'Best Practices', // 'Stylistic Issues', 'Deprecated Rules', 'Possible Error', 'Best Practices',
+      presets: {},
+      url:
+        'https://github.com/ember-template-lint/ember-template-lint/blog/master/docs/rules/inline-link-to.md',
```

Comment:
that URL leads to 404 for me

Reply:
I'm wondering if we should infer the URL from the rule name and `package.json` version instead 🤔